src/common/filesystembase.cpp

```diff
@@ -191,6 +196,11 @@ bool FileSystem::uncheckedRenameReplace(const QString &originFileName,
     }
 
 #else //Q_OS_WIN
+    if (FileSystem::isFileLocked(originFileName, FileSystem::LockMode::Exclusive)) {
+        *errorString = QCoreApplication::tr("FileSystem", "Can't rename %1, the file is currently in use").arg(originFileName);
```

Comment:
Same here, in this function `errorString` is used completely unchecked.

Reply:
I added an assert, as the function required an error parameter from the start